hw/ip/otbn/dv/otbnsim/sim/alert.py

```diff
@@ -9,12 +9,12 @@
 ERR_CODE_NO_ERROR = 0x0
 ERR_CODE_BAD_DATA_ADDR = 0x1
 ERR_CODE_BAD_INSN_ADDR = 0x2
-ERR_CODE_CALL_STACK = 0x3
-ERR_CODE_ILLEGAL_INSN = 0x4
-ERR_CODE_LOOP = 0x5
-ERR_CODE_FATAL_IMEM = 0x80
-ERR_CODE_FATAL_DMEM = 0x81
-ERR_CODE_FATAL_REG = 0x82
+ERR_CODE_CALL_STACK = 0x4
+ERR_CODE_ILLEGAL_INSN = 0x8
+ERR_CODE_LOOP = 0x10
+ERR_CODE_FATAL_IMEM = 0x20
+ERR_CODE_FATAL_DMEM = 0x40
+ERR_CODE_FATAL_REG = 0x80
```

Comment:
nit: I'd go for `(1 << x)` notation here to make cross-checking against the bits in the table slightly easier.

Reply:
I agree! Confusingly, I could have sworn I wrote that in a comment somewhere, but can't find it... Hmm.